docs/docker_quickstart.md

```diff
@@ -70,6 +70,40 @@ docker-compose up -d
 !!! Warning "Default configuration"
     While the configuration generated will be mostly functional, you will still need to verify that all options correspond to what you want (like Pricing, pairlist, ...) before starting the bot.
 
+#### Acessing the UI
+
+Uncommend the 2 lines below and add your IP adress in the following format (like 192.168.2.67:8080:8080) to the ft_userdata/docker-compose.yml:
+'''bash
+    ports:
+      - "yourIPadress:8080:8080"
+'''
+Your ft_userdata/user_data/config.json should look like:
+'''bash
+api_server": {
+        "enabled": true,
+        "listen_ip_address": "0.0.0.0",
+        "listen_port": 8080,
+        "verbosity": "error",
+        "enable_openapi": false,
+        "jwt_secret_key": "****",
+        "CORS_origins": [],
+        "username": "****",
+        "password": "****"
+    },
+'''
+instead of "****" you will have your data in.
+Then rebuild your docker file:
+Linux:
+'''bash
+sudo docker-compose down && sudo docker-compose pull && sudo docker-compose build && sudo docker-compose up -d 
```

Comment:
this is a ... dangerous command.

the build step will not work UNLESS you also have a custom docker file - so at this step of the documentation, i think this is wrong (the build step should only be in the section about custom dependencies).
will need to test if the build-step fails without build-section, or if it's simply ignored if no build is necessary.
It's however dangerous as the build-step will require an additional step (change the image name) - otherwise the pull step will not do anything assuming you have the "latest build" image already available.